backend/src/oauth/filter/errorPageException.filter.ts

```diff
@@ -7,6 +7,8 @@ export class ErrorPageExceptionFilter implements ExceptionFilter {
   constructor(private config: ConfigService) {}
 
   catch(exception: ErrorPageException, host: ArgumentsHost) {
+    console.error(exception);
```

Comment:
One minor thing, could you please the logger instead of a `console.error`? You can see an example here: https://github.com/stonith404/pingvin-share/blob/main/backend/src/jobs/jobs.service.ts#L12

Reply:
Ok, I'll fix it.